src/main/java/BasePage.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
+    protected WebDriver driver;
+
+    public BasePage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    // Enter text to text field
```

Comment:
In a remote repository there are should not be any comments. Please, remove all comments in your code.
You should have propper variable/method/class name for clear understanding why it is needed.

Reply:
Done